src/main/java/org/folio/service/BatchResourceService.java

```diff
@@ -0,0 +1,92 @@
+package org.folio.service;
+
+import java.util.List;
+import java.util.function.Function;
+
+import org.folio.rest.persist.PostgresClient;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import io.vertx.core.AsyncResult;
+import io.vertx.core.Future;
+import io.vertx.core.Handler;
+import io.vertx.ext.sql.SQLConnection;
+import io.vertx.ext.sql.UpdateResult;
+
+public class BatchResourceService {
```

Comment:
Has this class been copied from elsewhere?

Reply:
nope